qemu/tests/cfg/thin_provisioning_other.cfg

```diff
@@ -0,0 +1,28 @@
+- thin_provisioning_other:
```

Comment:
other means other cases since the case is based on the bug ,i didn't think of a suitable name ,so i name it other.